cosmo_tester/test_suites/broker_management/conftest.py

```diff
@@ -59,11 +67,13 @@ def _get_brokers(cfy, ssh_key, module_tmpdir, attributes, logger,
             node.verify_services_are_running = skip
             node.upload_necessary_files = skip
             node.upload_plugin = skip
-            node.enter_sanity_mode = skip
 
         hosts.create()
 
         for node_num, node in enumerate(hosts.instances, 1):
+            if node_num > broker_count:
```

Comment:
this is also used below as `for node in hosts.instances[:broker_count]:` so i'd do, above, `brokers = hosts.instances[:broker_count]` and use it here and below